src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs

```diff
@@ -303,7 +301,7 @@ public async Task<IActionResult> Create()
             // Creates a default shell settings based on the configuration.
             var shellSettings = _shellSettingsManager.CreateDefaultSettings();
 
-            var currentProfiles = shellSettings["FeatureProfile"]?.Split(ProfileSeparators);
+            var currentProfiles = shellSettings["FeatureProfile"]?.Split(',', StringSplitOptions.TrimEntries | StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Better I think to keep the 2 separators including the space as it allows to input `value1,value2` but also `value1, value2`, then `RemoveEmptyEntries` is no more necessary, as we do to split the tenant RequesturlHost in some places.

Reply:
@jtkech using the two separators will introduce a breaking change. Currently, a space can be part of the existing name which is stored in the settings. That would break. For example, if you have a profile with the name `test profile`, that would break the use if we trim using `',',' '` but using `.Split(',', StringSplitOptions.TrimEntries | StringSplitOptions.RemoveEmptyEntries)` will not be a breaking change.

Reply:
Ah okay if they are profiles names that may contain spaces, yes hosts don't have spaces, so maybe define a static string[] to split on `"," and ", "`, allowing at least one space after a comma.

Reply:
@jtkech I am now using `", "`

Reply:
@jtkech you good with the last changes?